src/rqt_top/node_info.py

```diff
@@ -40,57 +33,78 @@
 class NodeInfo(object):
     nodes = dict()
 
+    def __init__(self, node):
+        self._node = node
```

Comment:
Do we want to bypass the context wrapper? (https://github.com/ros-visualization/rqt/pull/161/) Couldn't we just pass in a copy of the context and then use the helper functions? Or maybe make `context._node` a public member variable (`context.node`) and remove the helper functions?

Reply:
I think this comment should go on the referenced PR